continuous_integration/runtests.sh

```diff
@@ -35,5 +35,5 @@ else
     if [ "$RUN_MEMORY" != "true" ]; then
         PYTEST_ARGS="$PYTEST_ARGS --skip-high-memory"
     fi
-    tox -v -e "${TOX_ENV}"  -- ${PYTEST_ARGS} --junitxml="${JUNITXML}"
+    tox -v -e "${TOX_ENV}"  -- ${PYTEST_ARGS} -x --junitxml="${JUNITXML}"
```

Comment:
Are we sure we want the `-x` option on the CI?

Reply:
It's was for faster iteration. I will undo.

Reply:
```suggestion
    tox -v -e "${TOX_ENV}"  -- ${PYTEST_ARGS} --junitxml="${JUNITXML}"
```